src/Sarif/HashData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
```

Comment:
Every file in this project has a copyright/license notice at the top. Copy/paste it from another file.